examples/tv-app/linux/main.cpp

```diff
@@ -114,24 +116,71 @@ class MyPincodeService : public PincodeService
+//                                         Controller::AccessControlCluster::ChangeTypeEnum changeType);
+
 class MyPostCommissioningListener : public PostCommissioningListener
 {
     void CommissioningCompleted(uint16_t vendorId, uint16_t productId, NodeId nodeId, OperationalDeviceProxy * device) override
     {
 
         // TODO:
-        // - the endpointId chosen should come from the App Platform (determined based upon vid/pid of node)
+        // - the videoPlayerEndpointId chosen should come from the App Platform (determined based upon vid/pid of node)
         // - the cluster(s) chosen should come from the App Platform
+        // - the contentAppEndpointId chosen should come from the App Platform (determined by which apps allow this vendorId)
+        // - the number of contentAppEndpointId could be 0 or more
+        // - the fabric index can't be 0, but needs to be validated
         constexpr EndpointId kBindingClusterEndpoint = 0;
 
-        GroupId groupId       = kUndefinedGroupId;
-        EndpointId endpointId = 1;
-        ClusterId clusterId   = kInvalidClusterId;
+        GroupId groupId                  = kUndefinedGroupId;
+        EndpointId videoPlayerEndpointId = 1;
+        EndpointId contentAppEndpointId  = 6;
+        ClusterId clusterId              = kInvalidClusterId;
+        FabricIndex fabricIndex          = 1;
+
+        ChipLogProgress(Controller, "Attempting to create ACL");
+
+        // size_t subjectIndex;
+        // size_t targetIndex;
+        Access::AccessControl::Entry::Target targets[] = {
+            { .flags    = Access::AccessControl::Entry::Target::kEndpoint,
+              .endpoint = contentAppEndpointId }, // grant access to the content app endpoint
+            { .flags    = Access::AccessControl::Entry::Target::kCluster | Access::AccessControl::Entry::Target::kEndpoint,
+              .cluster  = 0x050a,
+              .endpoint = videoPlayerEndpointId }, // 0x050a is content launcher
+        };
+
+        Access::AccessControl::Entry entry;
+        GetAccessControl().PrepareEntry(entry);
+        entry.SetAuthMode(Access::AuthMode::kCase);
+        entry.SetFabricIndex(fabricIndex);
+        entry.SetPrivilege(Access::Privilege::kOperate);
+        entry.AddSubject(nullptr, nodeId);
+        for (auto & target : targets)
+        {
+            entry.AddTarget(nullptr, target);
+        }
+
+        Access::AuthMode a;
+        FabricIndex f;
+        Access::Privilege p;
+        entry.GetAuthMode(a);
+        entry.GetFabricIndex(f);
+        entry.GetPrivilege(p);
+        ChipLogProgress(Controller, "Entry authmode=%d fabricIndex=%d privilege=%d", static_cast<int>(a), f, static_cast<int>(p));
+
+        GetAccessControl().CreateEntry(nullptr, entry, &fabricIndex);
+
+        // SubjectDescriptor subjectDescriptor = { .fabricIndex = fabricIndex, .authMode = AuthMode::kCase, .subject = nodeId };
```

Comment:
delete commented out code or add some description about why it is there (e.g. some todo explaining what the template code is supposed to be used for or when uncommenting is apropriate)

Reply:
that was WIP, not removed